wikitemplate-IPFS.md

```diff
@@ -87,68 +87,104 @@
+- [ ] Verify that clicking on the puzzle-piece icon on the browser toolbar, then `IPFS Companion`, will load a popup.  
+  - [ ] Click on the gears (settings) icon and confirm it loads the `Companion Preferences` page. 
+  - [ ] Click on `My Node` and confirm it opens the same ipfs-webui interface as `My Node` at `brave://ipfs-internals`
 
 ## IPFS URLs
 
 - [ ] Ensure each of the following IPFS URLs load over both `Gateway` and `Local node` modes:
-    - [ ] `ipfs://bafybeiemxf5abjwjbikoz4mc3a3dla6ual3jsgpdr4cjr3oz3evfyavhwq/wiki/Vincent_van_Gogh.html`
+    - [ ] `ipfs://bafybeiemxf5abjwjbikoz4mc3a3dla6ual3jsgpdr4cjr3oz3evfyavhwq/wiki/Vincent_van_Gogh.html#Life`
     - [ ] `ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi/`
     - [ ] `ipfs://QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/wiki/Tokyo_National_Museum.html`
     - [ ] `ipfs:QmYwAPJzv5CZsnA625s3Xf2nemtYgPpHdWEz79ojWnPbdG/readme`
 
 ## IPNS URLs
 
 - [ ] Ensure each of the following IPNS URLs load over both `Gateway` and `Local node` modes:
-   - [ ] `ipns://brantly.eth`
    - [ ] `ipns://en.wikipedia-on-ipfs.org`
-   - [ ] `ipns://libp2p.io/`
-   - [ ] `ipns://en.wikipedia-on-ipfs.org/wiki/Tokyo`
-   - [ ] `ipns://ipfs.io`
+   - [ ] `ipns://en.wikipedia-on-ipfs.org/wiki/Tokyo#Islands`
+   - [ ] `ipns://docs.ipfs.io`
+   - [ ] `ipns://brantly.eth` (ENS)
+   - [ ] `ipns://brave.crypto` (Unstoppable Domains)
 
-## Gateway
+## Gateway choice
+
+## The interstitial page
 
-- [ ] Verify, on a new profile, you can change the IPFS public gateway address to `https://cloudflare-ipfs.com/` via `Settings -> IPFS -> Change`.
 - [ ] Verify visiting `ipns://en.wikipedia-on-ipfs` loads the interstitial page.
-- [ ] Verify selecting `Use a public gateway` loads `https://cloudflare-ipfs.com/ipns/en.wikipedia-on-ipfs.org/wiki/`.
-- [ ] Verify, on a new profile, you can load `https://en.wikipedia-on-ipfs.org`, switch `Method to resolve IPFS resources` to either `Gateway` or `Local node` in `brave://settings/ipfs`, and then see an `Open using IPFS` badge/icon in the URL bar.
-- [ ] Verify clicking on `Open using IPFS` on `https://blog.ipfs.io/24-uncensorable-wikipedia` loads `ipfs://bafybeiaieqdmhtnehaau7kqoj2lmdfqc7juk34cjyb7dxr35vahp22bquu/24-uncensorable-wikipedia/`.
+- [ ] Verify selecting `Use a public gateway` loads `https://cf-ipfs.com/ipns/en.wikipedia-on-ipfs.org/wiki/` which then redirect to unique origin at `https://en-wikipedia--on--ipfs-org.ipns.cf-ipfs.com/wiki/` 
+- [ ] Verify selecting `Use local node` loads `ipns://en.wikipedia-on-ipfs.org/wiki/`  
+
+### Public Path gateway
+
+- [ ] Verify, on a new profile, you can't change the IPFS public gateway address to `https://cloudflare-ipfs.com/` via `Settings -> IPFS -> IPFS public gateway address
+ -> Change`.
+- [ ] An error informing that only gateways with proper Origin isolation can be used in browser context is displayed.
```

Comment:
This requires https://github.com/brave/brave-browser/issues/16999

Reply:
Same as above, re: would it make sense to add a note/reference to that issue, so we can keep track?